README.md

```diff
@@ -72,12 +72,11 @@ And the corresponding "date" function that returns a date for
 each data point.
 
 ```js
-var eventDropsChart = d3.chart.eventDrops();
+var eventDropsChart = d3.chart.eventDrops()
+                              .date(d => d.date);
```

Comment:
@jpetitcolas If you want to replace a space by a space, I would react just as @sedy-bot!